src-built-in/components/windowTitleBar/src/stores/windowTitleBarStore.js

```diff
@@ -14,11 +14,17 @@ let constants = {};
 var Actions = {
 	initialize: function () {
 		// This ensures that our config is correct, even if the developer missed some entries
-		var options = FSBL.Clients.WindowClient.options;
+		let options = FSBL.Clients.WindowClient.options;
 		// TODO, this should come from config server (probably via a WindowsClient.getConfig() command), so that live components always have the latest config. Currently this config gets saved with the component via workspace customData.
-		var windowTitleBarConfig = options.customData.foreign.components["Window Manager"];
-		var FSBLHeader = windowTitleBarConfig.FSBLHeader;
-		var self = this;
+		let windowTitleBarConfig = options.customData.foreign.components["Window Manager"];
+		let FSBLHeader = windowTitleBarConfig.FSBLHeader;
+		let self = this;
+		let displayName = null;
+
+		//Display name is first up for display. If it doesn't exist, we look for a default title (or one that's already set on init). If no title, we display the window's name (e.g., '5434-Welcome Component-2321').
```

Comment:
Since title has higher priority for the titlebar this comment should be updated.

Reply:
Adding/removing comments, renaming variables, was not within the scope of this PR

Reply:
+1 please update comment so it makes sense